packages/time-picker/src/TimePicker/TimePicker.tsx

```diff
@@ -0,0 +1,202 @@
+  status?: StatusType
+  validationMessage?: React.ReactNode
+}
+
+export const TimePicker: React.VFC<TimePickerProps> = ({
+  status = "default",
+  validationMessage,
+  id,
+  isDisabled,
+  value,
+  onChange,
+  label,
+  timeZone = getLocalTimeZone(),
+  locale,
+  dropdownIncrements,
+
+  ...restProps
+}: TimePickerProps) => {
+  const handleOnChange = (timeValue: TimeValue): void => {
+    // onChange does not fire until user interacts with all placeholders
+    // if user interacts with spin buttons, timeValue is Time type, which cannot be converted directly into Date object
+    // if user interacts with MenuItems, it has to be CalendarDateTime or ZonedDateTime, as Time objects cannot be formatted properly to display in a list
+    // console.log("STATE", state.dateValue, timeValue)
+    if (isZonedDateTimeObject(timeValue)) {
+      onChange(timeValue.toDate())
+    } else {
+      const today = now(timeZone).toDate()
+      const date = convertTimeToZonedDateTime({
+        date: value ?? today,
+        hour: timeValue.hour,
+        minutes: timeValue.minute,
+        timeZone,
+      }).toDate()
+      onChange(date)
+    }
+  }
+
+  const state = useTimeFieldState({
+    ...restProps,
+    // @ts-expect-error controlled values should not be undefined and library throws warning if such is supplied for value,
+    // however library does not allow for null values despite handling properly
+    // https://github.com/adobe/react-spectrum/blob/main/packages/%40react-stately/utils/src/useControlledState.ts#L23
+    // https://github.com/adobe/react-spectrum/blob/main/packages/%40react-stately/datepicker/src/useTimeFieldState.ts
+    value: value ? parseAbsolute(value.toISOString(), timeZone) : null,
+    onChange: handleOnChange,
+    isDisabled,
+    hideTimeZone: true,
+    locale,
+    validationState: status === "default" ? "valid" : "invalid",
+  })
+  const menuState = useMenuTriggerState({})
+
+  const inputRef = React.useRef(null)
+  const { fieldProps } = useTimeField({ ...restProps, label }, state, inputRef)
+
+  const { menuProps, menuTriggerProps } = useMenuTrigger<TIME_OPTION>(
+    { isDisabled },
+    menuState,
+    inputRef
+  )
+
+  const timeZoneLabel = Intl.DateTimeFormat(locale, {
+    timeZoneName: "short",
+    timeZone,
+  })
+    .formatToParts(new Date())
+    .find(segment => segment.type === "timeZoneName")?.value
+
+  const options = useMemo(
+    () =>
+      getAllTimeOptions({
+        locale,
+        timeZone,
+        date: value,
+        increments: dropdownIncrements,
+      }),
+    [locale, timeZone, dropdownIncrements]
+  )
+
+  return (
+    <>
+      <Label data-testid="timepicker-label">{`${label} ${
+        restProps.hideTimeZone && timeZoneLabel ? "" : `(${timeZoneLabel})`
+      }`}</Label>
+      <div className={styles.wrapper}>
+        {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events */}
+        <div
```

Comment:
Is there a reason this isn’t a `<button>`?

Reply:
There's been a few conversations about this, and I think there were a few reasons why this is a `div` - the primary reason I believe is because we wanted the dropdown chevron to also be a button

Reply:
Returning back to this - @HeartSquared talked to dug about this, and confirmed we want to keep this a div, as turning it into a button may remove all the accessibility information from inside it.